upgrade_test.py

```diff
@@ -529,6 +528,11 @@ def test_upgrade_cql_queries(self):
         Run a set of different cql queries against various types/tables before
         and after upgrade of every node to check the consistency of data
         """
+        if self.params.get('upgrade_node_system'):
+            InfoEvent(message='Upgrading system on nodes').publish()
+            parallel_obj = ParallelObject(objects=self.db_cluster.nodes, timeout=self.system_upgrade_timeout)
+            parallel_obj.run(self.upgrade_system)
```

Comment:
it's repeated 5 times, couldn't we move it into `upgrade_system`?
also, maybe rename it to `upgrade_os`?

Reply:
Changed that